src/main/java/ca/bc/gov/chefs/etl/core/json/Form.java

```diff
@@ -5,6 +5,13 @@
 
 @JsonIgnoreProperties(ignoreUnknown=true)
 public class Form{
+    @Override
```

Comment:
I prefer to have toString() methods at the end of the file.

Reply:
toString() is now the last method in data models.